Apps/Playground/App.tsx

```diff
@@ -74,16 +75,42 @@ const EngineScreen: FunctionComponent<ViewProps> = (props: ViewProps) => {
     }
   }, [rootNode, scale]);
 
+  const trackingStateToString = (trackingState: WebXRTrackingState | undefined) : string =>
+  {
+    if (trackingState == undefined) {
+      return ""
+    }
+
+    switch(trackingState) {
+      case WebXRTrackingState.NOT_TRACKING:
+        return "Not Tracking";
+        break;
+      case WebXRTrackingState.TRACKING:
+        return "Tracking";
+        break;
+      case WebXRTrackingState.TRACKING_LOST:
+        return "Tracking Lost";
+        break;
+    }
```

Comment:
I would just do this:
```suggestion
    return trackingState === undefined ? "" : WebXRTrackingState[trackingState];
```
That will give you strings like "NOT_TRACKING" but I think that is fine. Your call, but I think the simpler code that results in slightly less polished state string is a bit better.

Reply:
I like that a lot! I wasn't aware of that syntax.